pubspec.yaml

```diff
@@ -0,0 +1,6 @@
+name: wCommon
+description: Dart bindings for wf-common
+dependencies:
+dev_dependencies:
+  unittest: '0.11.5+4'
```

Comment:
We should probably not include the "+4" since build meta data shouldn't determine precedence: 

> 10. Build metadata MAY be denoted by appending a plus sign and a series of dot separated identifiers immediately following the patch or pre-release version. Identifiers MUST comprise only ASCII alphanumerics and hyphen [0-9A-Za-z-]. Identifiers MUST NOT be empty. Build metadata SHOULD be ignored when determining version precedence. Thus two versions that differ only in the build metadata, have the same precedence. Examples: 1.0.0-alpha+001, 1.0.0+20130313144700, 1.0.0-beta+exp.sha.5114f85.

http://semver.org